core/store/src/db.rs

```diff
@@ -46,6 +46,10 @@ impl Into<io::Error> for DBError {
     }
 }
 
+/// Number of DBCol columns.
+/// Do not move this line from enum DBCol.
```

Comment:
I feel the warning is outdated

Reply:
@frol I feel the same way. Regardless, of the warning, the variable got moved anyway.

I removed the warnings, as it serves no purpose.